src/cluster.h

```diff
@@ -66,7 +66,7 @@ int clusterAllowFailoverCmd(client *c);
 void clusterPromoteSelfToMaster(void);
 int clusterManualFailoverTimeLimit(void);
 
-void clusterCommandSlots(client * c);
+void clusterSlotsCommand(client * c);
```

Comment:
Is there any reason for changing the method name?

Reply:
Did I suggest this change? My bad. I think the norm was actually `clusterCommand*`. Sorry.